meson.build

```diff
@@ -182,6 +182,15 @@ conf.set_quoted('SYSTEM_SYSVINIT_PATH',                       sysvinit_path)
 conf.set_quoted('SYSTEM_SYSVRCND_PATH',                       sysvrcnd_path)
 conf.set_quoted('RC_LOCAL_SCRIPT_PATH_START',                 get_option('rc-local'))
 conf.set_quoted('RC_LOCAL_SCRIPT_PATH_STOP',                  get_option('halt-local'))
+
+if get_option('ok-color').to_lower() == 'green'
+        conf.set_quoted('ANSI_OK_COLOR', '\x1B[0;32m')
```

Comment:
So... you removed *two* levels of indirection. Sorry, my earlier comment wasn't clear in this regard. Please keep one level of indirection, i.e. do
```python
conf.set('ANSI_OK_COLOR', 'ANSI_COLOR_' + get_option('ok-color').to_upper())
```
The reason is that we don't want to duplicate the ansi sequence here, once in the header file is enough.

Reply:
Thanks, this looks sensible. I'm new to meson. Will update the code.